apps/web/components/molecules/pagination/pagination.tsx

```diff
@@ -0,0 +1,107 @@
+import { ChevronLeftIcon, ChevronRightIcon } from '@heroicons/react/20/solid';
+import clsx from 'clsx';
+import { VisuallyHidden } from 'ui';
+import { useRouter } from 'next/router';
+
+type PaginationProps = {
+	quantity: number;
+};
+
+export const Pagination = ({ quantity }: PaginationProps) => {
+	const router = useRouter();
+	const currentPage: number =
+		router.query.page && !isNaN(Number(router.query.page))
+			? Number(router.query.page)
+			: 1;
+	const perPage =
+		router.query.page && !isNaN(Number(router.query.limit))
+			? Number(router.query.limit)
+			: 11;
+	const totalPages = Math.ceil(quantity / perPage);
+
+	return (
+		<div className="flex items-center justify-between border-t border-gray-200 bg-white px-4 py-3 sm:px-6">
+			<div className="flex flex-1 justify-between sm:hidden">
+				<a
+					href={`?page=${currentPage - 1}`}
+					className="relative inline-flex items-center rounded-md border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 hover:bg-gray-50"
+				>
+					Previous
+				</a>
+				<a
+					href={`?page=${currentPage + 1}`}
+					className="relative z-10 inline-flex items-center bg-blue-600 px-4 py-2 text-sm font-semibold text-white focus:z-20 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-600"
+				>
+					{currentPage}
+				</a>
+				<a
+					href="#"
+					className="relative ml-3 inline-flex items-center rounded-md border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 hover:bg-gray-50"
+				>
+					Next
+				</a>
+			</div>
+			<div className="hidden sm:flex sm:flex-1 sm:items-center sm:justify-between">
+				<div>
+					<p className="text-sm text-gray-700">
+						<span className="font-medium">
+							{(currentPage - 1) * perPage + 1}
+						</span>
+						-
+						<span className="font-medium">
+							{currentPage !== totalPages ? currentPage * perPage : quantity}
```

Comment:
What's that? 😮

Reply:
What is this comment? Explain your thoughts please, give others a chance to learn